src/components/model.js

```diff
@@ -90,4 +99,7 @@ module.exports.Component = registerComponent('loader', {
       el.setObject3D('mesh', collada.scene);
     });
   }
-});
+};
+
+module.exports.Component = registerComponent('loader', ModelComponent);
+module.exports.Component = registerComponent('model', ModelComponent);
```

Comment:
this things will pollute the code and will won't remember to remove it. I think there are not that many examples out there so I'm not concerns with backwards compatibility